docker/Dockerfile

```diff
@@ -1,14 +1,18 @@
 # BUILD STAGE ---------------------------------------
 # split this stage to save time and reduce image size
 # ---------------------------------------------------
-FROM python:3.10-bullseye as BuildStage
```

Comment:
Hi @devine12 thanks for the contribution, can you create a folder community inside the docker folder and add there your changes?
We would love to add it but we don't want to break it for people that using this docker and don't want to update their base image (no internet connection etc..)

Reply:
Alternatively, the base image can be a docker build ARG.
Then we can build & publish both for each version